Detectors/EMCAL/simulation/src/Digitizer.cxx

```diff
@@ -21,6 +21,8 @@
 #include <forward_list>
 #include <chrono>
 #include <TRandom.h>
+#include "TF1.h"
+#include "TMath.h"
```

Comment:
generally, it is preferable to use math function from cmath in their `std::pow` instead of `pow` form, etc. Use ROOT if cmath doesn't provide a certain function (for instance Gamma).